pyro/infer/traceenum_elbo.py

```diff
@@ -11,10 +12,9 @@
 import pyro.poutine as poutine
 from pyro.distributions.util import is_identically_zero
 from pyro.infer.elbo import ELBO
-from pyro.infer.enum import iter_discrete_traces
+from pyro.infer.enum import _get_importance_trace, _iter_discrete_escape, _iter_discrete_extend
```

Comment:
nit: If we're importing these across modules, I think it makes sense to remove the leading underscore.